spring-cloud-dataflow-core/src/test/java/org/springframework/cloud/dataflow/core/dsl/StreamParserTests.java

```diff
@@ -451,10 +432,10 @@ public void errorCases06() {
 
 	@Test
 	public void errorCases07() {
-		checkForParseError("foo > bar", DSLMessage.EXPECTED_CHANNEL_PREFIX_QUEUE_TOPIC, 6, "bar");
-		checkForParseError("foo >", DSLMessage.OOD, 5);
-		checkForParseError("foo > --2323", DSLMessage.EXPECTED_CHANNEL_PREFIX_QUEUE_TOPIC, 6, "--");
-		checkForParseError("foo > *", DSLMessage.UNEXPECTED_DATA, 6, "*");
+		//checkForParseError("foo > bar", DSLMessage.EXPECTED_CHANNEL_PREFIX, 6, "bar");
+		checkForParseError(":foo >", DSLMessage.OOD, 6);
+		checkForParseError(":foo > --2323", DSLMessage.EXPECTED_MODULENAME, 7, "--");
```

Comment:
this is not testing the same situation as the one it modified (that was a missing prefix error)